### PR TITLE
Add implementation pointer to ign-common classes

### PR DESCRIPTION
**Pull request by luca-della-vedova**
To be merged after #160, for now only classes in `graphics` have been refactored, rest of the library will follow.

### CODE REVIEW

**Comment by mjcarroll** [comment]
Thanks @luca-della-vedova for getting this cleaned up!

File: graphics/src/STLLoader.cc [hunk 1]
```diff
@@ -28,9 +28,16 @@
 using namespace ignition;
 using namespace common;
 
+
+//////////////////////////////////////////////////
+class ignition::common::STLLoader::Implementation
+{
+};
+
```
Comment (mjcarroll) [inline graphics/src/STLLoader.cc:36]: Since this one is empty was this some sort of scripting/automation thing?

Comment (luca-della-vedova) [reply graphics/src/STLLoader.cc:36]: Not really I just went and added implementation pointers to all the classes, the main idea was that if in the future we had to add any data member it wouldn't cause API breakage but I'm not sure how likely the case is for the `STLLoader` class

Comment (mjcarroll) [reply graphics/src/STLLoader.cc:36]: Ah this makes more sense, good call. May not be universally useful, but does potentially save some headaches later.

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**